scripts/merge-repos.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+import yaml
+
+# Load YAML repos files
+with open('ros2.repos', 'r') as ros2_file:
```

Comment:
probably it will be the only place where we use this file, but it would be cool to parametrize ros2.repos and spaceros.repos